src/components/layout/navigation/Sorting.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import MenuItem from '@mui/material/MenuItem';
+import Select, {SelectChangeEvent} from '@mui/material/Select';
+import Box from "@mui/material/Box";
+import {Typography} from "@mui/material";
+
+export const Sorting = () => {
+    const [value, setValue] = React.useState('1');
+    const sort = ['Verify', 'Date', 'Popularity', 'Something else']
+
+    const handleChange = (event: SelectChangeEvent) => {
+        setValue(event.target.value);
+    };
+
+    return (
+        <Box sx={{
+            display: 'flex',
+            justifyContent: 'end',
+            alignItems: 'center',
+            gap: 2.5
+        }}>
+            <Typography sx={{
+                typography: 'subtitle1',
+                color: 'text.primaryAlpha300'
+            }}>
+                Sorting
+            </Typography>
+            <Select
+                sx={{
+                    minWidth: 120,
+                    height: 26,
+                    bgcolor: 'primary.contrastText',
+                    borderRadius: '10px',
+                    typography: 'subtitle1',
+                    '& .MuiOutlinedInput-notchedOutline': {
+                        border: 0
```

Comment:
need transfer to MUI theme, because this is a common style for all our selectors

Reply:
don't for all